data-es2016plus.js

```diff
@@ -619,11 +619,12 @@ exports.tests = [
         name: 'no line break between async and function',
         exec: function () {/*
           async function a(){}
-          try { Function("async\n function a(){}")(); } catch(e) { return true; }
+          try { Function("async\n function a(){await 0}")(); } catch(e) { return true; }
```

Comment:
I updated this test, because I don't think that the original one worked?
```js
async
function a(){}
```
is valid code in non-strict mode (or in strict mode if someone defined `globalThis.async`).